website/docs/cli/commands/help.mdx

```diff
@@ -12,6 +12,14 @@ import Terminal from '@site/src/components/Terminal'
 ## Usage
 
 The `atmos --help` and `atmos -h` commands show help for all Atmos CLI commands.
+Please note that an Atmos upgrade version box is shown at the end of each help command.
+The frequency of the update box message is configured in the atmos.yaml file.
+There are six frequency options: minute, hourly, daily, weekly, monthly, and yearly.
+The default value is set to daily.
+You can disable this feature in the atmos.yaml file by setting enable: false,
+or by setting the environment variable ATMOS_VERSION_CHECK_ENABLED=false,
+which overrides the config's setting.
```

Comment:
From time to time, Atmos will check for a newer release and let you know if one is available. Please see the documentation for Atmos version to configure this behavior.

Reply:
Let's not repeat all this redundant information here and instead instruct them to go to the version page

Reply:
just changed and pushed new text for help.mdx